src/vs/vscode.d.ts

```diff
@@ -1123,15 +1123,16 @@ declare module 'vscode' {
 		/**
 		 * Derive a new Uri from this Uri.
 		 *
+		 * ```ts
```

Comment:
I moved this out of a @sample tag as it looked really bad on the website https://code.visualstudio.com/docs/extensionAPI/vscode-api#Uri.with